Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.8
+
+WORKDIR /app
+
+RUN apk add --no-cache wget ca-certificates && \
+    wget -O /etc/apk/keys/phpearth.rsa.pub https://repos.php.earth/alpine/phpearth.rsa.pub && \
+    echo "https://repos.php.earth/alpine/v3.7" >> /etc/apk/repositories && \
+    apk add --no-cache --update certbot nginx dcron curl tini php7.2 php7.2-bcmath php7.2-common php7.2-dom php7.2-fpm php7.2-gd php7.2-mbstring php7.2-openssl php7.2-zip php7.2-pdo php7.2-phar php7.2-json php7.2-pdo_mysql php7.2-session php7.2-ctype php7.2-tokenizer php7.2-zlib php7.2-simplexml && \
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

Comment:
I would replace the section above with this and thereby remove the dependency on a third party repo:
```
+RUN apk update \
    && apk add --no-cache \
        ca-certificates \
        certbot \
        composer \
        curl \
        dcron \
        nginx \
        php7 \
        php7-bcmath \
        php7-common \
        php7-ctype \
        php7-dom \
        php7-fpm \
        php7-gd \
        php7-json \
        php7-mbstring \
        php7-openssl \
        php7-pdo \
        php7-pdo_mysql \
        php7-phar \
        php7-session \
        php7-simplexml \
        php7-tokenizer \
        php7-zip \
        tini \
        wget
```

Reply:
Does that install 7.2 though?